runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/AutocompleteWidget.tsx

```diff
@@ -3,8 +3,9 @@ import React, { FC, useState } from 'react';
 import { Popper, PopperProps, TextField, Autocomplete } from '@mui/material';
 import { WidgetProps } from '@rjsf/core';
 
-const AutocompleteWidget: FC<WidgetProps & { groupBy?: (option: any) => string }> = (props) => {
+const AutocompleteWidget: FC<WidgetProps & { groupBy?: (option: any) => string } & { customError?: string[] }> = (props) => {
```

Comment:
Move this whole type to an interface and then move this interface to seperate file

Reply:
done